gobblin-core/src/main/java/gobblin/source/extractor/extract/jdbc/JdbcExtractor.java

```diff
@@ -943,6 +970,19 @@ public long getCount(CommandOutput<?, ?> response) throws RecordCountException {
       throw new DataRecordException("Failed to get records from MySql; error - " + e.getMessage(), e);
     }
   }
+  
+  private void initialBlobTypeHashSet(Set<String> input, ResultSetMetaData resultsetMetadata) throws SQLException{
```

Comment:
nit: it looks more natural for the function to just return the set rather than have it passed as a parameter.